src/System.Diagnostics.TextWriterTraceListener/tests/DelimiterWriteMethodTests.cs

```diff
@@ -8,14 +8,14 @@
 
 namespace System.Diagnostics.TextWriterTraceListenerTests
 {
-    public class DelimiterWriteMethodTests : IDisposable
+    public class DelimiterWriteMethodTests : FileCleanupTestBase
     {
         private readonly Stream _stream;
         private readonly string _fileName;
 
         public DelimiterWriteMethodTests()
         {
-            _fileName = string.Format("{0}.xml", GetType().Name);
+            _fileName = $"{GetTestFilePath(null, GetType().Name)}.xml";
```

Comment:
I think just GetTestFilePath() should be enough: the directory already contains `GetType().Name` and the file contains  `[CallerMemberName]`